documentation/examples/remote_storage/remote_storage_adapter/influxdb/client.go

```diff
@@ -64,6 +65,8 @@ func NewClient(logger log.Logger, conf influx.HTTPConfig, db string, rp string)
 			},
 		),
 	}
+	go client.ensureDatabase()
```

Comment:
shouldn't this be blocking or do some additional check in the Write,Read funcs to make sure we don't try to read or read until the db is ready?

Reply:
done.
(I was doing this intentionally to make remote_storage_adaptor not to wait influxdb.)